### DISCUSSION TITLE
120Hz on PSM Galaxy S23 ONEUI6.1

### DISCUSSION THREAD

**Discussion by sonicpeter** [tribalfs/GalaxyMaxHzPub#229] [Q&A] [answered]
Dear all, 

App installed by command "adb install --bypass-low-target-sdk-block......." .

"Allow restricted settings" not found in "Settings > Apps > Galaxy MaxHz > Click top right menu and Allow restricted settings." since there is only one option in top right : Uninstall for all users

I also went to Accessibility and Galaxy MaxHz is ON already. 

What's more, I also executed the following command:
adb shell pm grant com.tribalfs.gmh android.permission.WRITE_SECURE_SETTINGS

Then, worse still, all button in the app can be chosen but nothing works at all. 

I have work around by using App Opps too to allow all necessary permission to try my best to make it work but nothing helped... So, what I guess is Samsung did something shit on the system. 
 
![image](https://github.com/user-attachments/assets/95cf7a22-2c57-4299-b4f0-a608bbef51d5)

If it works finally, I will buy a coffee for sure. Thank you very much.

**Comment by tribalfs** [comment, marked as answer]
"I also went to Accessibility and Galaxy MaxHz is ON already."
- If it's already ON, it should be good.  GMH settings should already stick.
 

"All button in the app can be chosen but nothing works at all."
- If you are referring to setting refresh rate to higher than 60hz, you have to disable the _Power Saving Mode_ or perform the extra steps provided in the app /[app description](https://github.com/tribalfs/GalaxyMaxHzPub#user-content-fn-3-7de0d0adfa0ebec466939179aa63138b)  if you want it turned on.

**Reply by sonicpeter** [reply]
Great. It works now using extra steps! I love it. 

However, the banking apps all failed to use when MaxHz app exists. Those apps request me to delete your app before using their apps... I think there is no means to get rid of it ? Thanks

**Comment by Onewpus** [comment]
On the latest version of the Galaxy Max Hz, my S23 Ultra has long had Double Tap errors to open the screen. I hope admin will answer

**Reply by tribalfs** [reply]
It's unclear what you mean; please provide more details. Also please open new  discussion if not related to any existing discussions.
